### DISCUSSION TITLE
heroku-like button

### DISCUSSION THREAD

**Discussion by makayabou** [dokku/dokku#4327] [Q&A]
Hello,
Could someone point me to some doc / example to create a heroku-like button (the one that links to https://heroku.com/deploy?template=https://github.com/my-app) , linked to my own dokku instance ?

Thanks

**Comment by josegonzalez** [comment]
There isn't a built in way to do this:

- We don't provide any web api with the core project
- There isn't (currently) a way to build projects from an arbitrary repo/commit. #4297 may handle that

That said, I'm working on a (paid) api/git server/web ui in this space, so this is definitely a cool idea for that.
